### DISCUSSION TITLE
Search filter button

### DISCUSSION THREAD

**Discussion by worldlydev** [wstyres/Zebra#923] [Ideas]
Would be cool if the new search filter buttons could be hidden until you press the filter button like in Sileo.

![4CA3E4AD-9AD5-44BF-BA42-3A8E22C950E5](https://user-images.githubusercontent.com/51668635/75617428-30fb0780-5b5f-11ea-9ca0-a4a8ad2e741b.jpeg)
![C4F0A338-4328-417F-9B88-22766D81A6B2](https://user-images.githubusercontent.com/51668635/75617430-33f5f800-5b5f-11ea-83aa-b0c4aa2ed265.jpeg)

Screen recording:
https://imgur.com/a/IUasiJn

![A16AFE23-EDE2-44D4-859F-6CAA6422C5C7](https://user-images.githubusercontent.com/51668635/75617442-59830180-5b5f-11ea-8d7d-7db551f48049.jpeg)

Perhaps the search bar let's you search though name, developer, and description all at once and you can choose to filter for one of those specifically with a filter button.

This way you can search through everything all at once unless you are looking for something more specific.

**Comment by wstyres** [comment]
We're still working on the UI for this so we'll take this into account.
